### DISCUSSION TITLE
Are LEFT OUTER JOIN's Possible??

### DISCUSSION THREAD

**Discussion by Dreamystify** [prisma/prisma#5745] [Q&A]
I have a graphql server using prisma, and I'm trying to circumvent the N + 1 issue. I have a table of purchase orders which each purchase order is connected to an agent. Im trying to retrieve all the purchase orders and get the agent name for the purchase order via an agent_id on the purchase order. This can be done with a left outer join but Im not sure how to do it with prisma. The following I've tried but doesn't work
```ts
    return await prisma.purchase_orders.findMany({
        where: {
            active: true,
        },
        include: {
             agents: true,
        },
    }); 
```
A sample of the sql that works is:
```sql
    SELECT p.order_id, a.agent_id, a.name FROM purchase_orders as p
    LEFT OUTER JOIN agents as a ON p.agent_id = a.agent_id;
```
Any advice would be appreciated

**Comment by ryands17** [comment]
@Dreamystify 👋 

Prisma already circumvents the N+1 issue as described in [this](https://www.youtube.com/watch?v=7oMfBGEdwsc&amp;ab_channel=Prisma) talk.

As for the above query, if you just want to get the `agents` for the `purchase_orders` then the above query with `include` should work fine. What's the issue you're facing?

**Reply by Dreamystify** [reply]
Turns out, it was how i structured the models to call prisma in the resolvers, if I set it up like in the talk it does indeed work as expected

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
